/data/run01/scv7454/GraduationProject/Library/PhysModule.py:121: UserWarning: Casting complex values to real discards the imaginary part (Triggered internally at  ../aten/src/ATen/native/Copy.cpp:244.)
  mag[s, n] = tc.einsum('ab,ba->', rho.type(tc.complex64), which_ops[s].type(tc.complex64))
/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages/matplotlib/cbook/__init__.py:1335: ComplexWarning: Casting complex values to real discards the imaginary part
  return np.asarray(x, float)
['/data/run01/scv7454/GraduationProject', '/data/home/scv7454/.conda/envs/myname/lib/python39.zip', '/data/home/scv7454/.conda/envs/myname/lib/python3.9', '/data/home/scv7454/.conda/envs/myname/lib/python3.9/lib-dynload', '/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages', '/data/run01/scv7454/GraduationProject\\Library']
Traceback (most recent call last):
  File "/data/home/scv7454/run/GraduationProject/TimeEvolution.py", line 191, in <module>
    time_evolution_Heisenberg_chain(para, state)
  File "/data/home/scv7454/run/GraduationProject/TimeEvolution.py", line 147, in time_evolution_Heisenberg_chain
    bf.output_txt(mag_z_tot, filename='GraduationProject/Data/mag_z_dt{:d}.txt'.format(para['print_time_it']))
  File "/data/run01/scv7454/GraduationProject/Library/BasicFun.py", line 285, in output_txt
    np.savetxt(filename, x)
  File "/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages/numpy/lib/npyio.py", line 1556, in savetxt
    open(fname, 'wt').close()
FileNotFoundError: [Errno 2] No such file or directory: 'GraduationProject/Data/mag_z_dt10.txt'
设置参数
Traceback (most recent call last):
  File "/data/home/scv7454/run/GraduationProject/ADQC.py", line 161, in <module>
    states = np.load('GraduationProject/Data/states_dt{:d}_tot{:.0f}.npy'.format(interval, args.time), allow_pickle=True)
  File "/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages/numpy/lib/npyio.py", line 427, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GraduationProject/Data/states_dt10_tot100.npy'
Traceback (most recent call last):
  File "/data/home/scv7454/run/GraduationProject/1toAll.py", line 18, in <module>
    qc = tc.load('GraduationProject/Data/qc_dt{:d}_tot{:.0f}.pth'.format(interval, time_tot))
  File "/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/data/home/scv7454/.conda/envs/myname/lib/python3.9/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'GraduationProject/Data/qc_dt10_tot100.pth'
